app/konnect/org-management/okta-idp.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Set Up Okta IdP Authentication
+no_version: true
+badge: enterprise
+---
+
+As an alternative to Konnect Cloud’s native basic authentication, you can set up
+org access through Okta. This way, your users can log in to Konnect Cloud
+using their Okta credentials, and without needing a separate login.
+
+You can't mix authenticators in Konnect Cloud. With Okta IdP
+authentication enabled, all non-admin Konnect users will subsequently log in
+through Okta. Only Konnect admins can continue to log in with basic
+authentication.
+
+> This feature is only available with Kong Konnect Enterprise.
+
+## Prerequisites
+* [**Organization Admin**](/konnect/reference/org-management/#role-definitions)
+permissions in Konnect Cloud
+* [Okta developer account](https://developer.okta.com/)
```

Comment:
I would say you need access to the Okta configuration for your organization. In our demos we say you need Okta developer access because we've been using a sandbox environment for Okta. However, customers like Vanguard will need to have access to their production Okta environment (or will need to work with the team that does).